### PR TITLE
User story32

### PR DESCRIPTION
**Pull request by KellyIB**
added model tests for new model methods, feature testing, new routes, user controller in the admin folder, and model methods.  resolved merge conflicts

### CODE REVIEW

File: app/models/order.rb [hunk 1]
```diff
@@ -23,12 +23,29 @@ def item_quantity_merchant(merchant)
...
   end
 
   def status_changer
     if item_orders.all? { |itemorder| itemorder.status == 'fulfilled' }
       update(status: 'packaged')
     end
   end
+
+  def self.pending_status
+    all.where(status: "pending")
```
Comment (adumortier) [inline app/models/order.rb:36]: nice and concise use of AR here

File: app/views/admin/dashboard/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,43 @@
+<center>
+<h1>Admin Dashboard</h1>
+</center>
+
+  <section class= "packaged-orders">
```
Comment (adumortier) [inline app/views/admin/dashboard/index.html.erb:5]: super well organized and easy to read

File: spec/features/admin/dashboard_spec.rb [hunk 3]
```diff
@@ -0,0 +1,91 @@
+require 'rails_helper'
+
+  describe "As an admin_user" do
+    describe "I visit my dashboard" do
+      before :each do
+        @user1 = create(:random_user)
```
Comment (adumortier) [inline spec/features/admin/dashboard_spec.rb:6]: nice job on making use of FactoryBot

File: spec/models/order_spec.rb [hunk 4]
```diff
@@ -16,41 +16,71 @@
...
+    it "can find shipped orders" do
+      expect(Order.shipped_status).to eq([@order4, @order6])
+    end
 
-    expect(@order_1.status).to eq("packaged")
+    it "can find cancelled orders" do
+      expect(Order.cancelled_status).to eq([@order7])
     end
   end
 end
```
Comment (TravisBorgsmiller) [inline spec/models/order_spec.rb:86]: Once again love the model tests!

File: config/routes.rb [hunk 5]
```diff
@@ -42,11 +42,12 @@
...
-      get '/', to: 'dashboard#index'
-      get '/merchants', to: 'merchants#index'
-      get '/merchants/:id', to: 'merchants#show'
-    end
+  namespace :admin do
+    get '/', to: 'dashboard#index'
+    get '/users/:user_id', to: 'users#show'
+    get '/merchants', to: 'merchants#index'
+    get '/merchants/:id', to: 'merchants#show'
+  end
```
Comment (TravisBorgsmiller) [inline config/routes.rb:51]: Routes looking good!

**Review by TravisBorgsmiller — APPROVED** [review]

**Review by adumortier — APPROVED** [review]

**Pull request merged**